app/src/main/java/com/duckduckgo/app/browser/newtab/NewTabLegacyPageViewModel.kt

```diff
@@ -61,7 +61,7 @@ import kotlinx.coroutines.withContext
 @ContributesViewModel(ViewScope::class)
 class NewTabLegacyPageViewModel @Inject constructor(
     private val dispatchers: DispatcherProvider,
-    private val remoteMessagingModel: Lazy<RemoteMessagingModel>,
+    private val remoteMessagingModel: RemoteMessageModel,
```

Comment:
Not sure if this should be Lazy, potentially yes coming from https://app.asana.com/0/488551667048375/1205234536443750/f cc/ @cmonfortep just in case.